"use strict";
class ValidationError extends Error {
    constructor(mensagem) {
        super(mensagem);
        this.name = 'ValidationError';
    }
}
function vaiDarErro() {
    throw new ValidationError('Dados inv√°lidos');
}
try {
    vaiDarErro();
}
catch (erro) {
    console.log(erro.name);
    console.log(erro.menssage);
}
//# sourceMappingURL=index.js.map